js/common.js

```diff
@@ -231,12 +231,11 @@ $(document).ready( function() {
         enableDateTimePicker(this);
 	});
 
-	if( $( ".dropzone-form" ).length ) {
-		enableDropzone( "dropzone", false );
-	}
-	if( $( ".auto-dropzone-form" ).length ) {
-		enableDropzone( "auto-dropzone", true );
-	}
+	$( 'form .dropzone' ).each(function(){
+		var classPrefix = 'dropzone';
```

Comment:
Do we need this variable vs. just using the string directly?  Why not have the class hard-coded in enableDropzone()?